lib/plenario_web/controllers/admin/meta_controller.ex

```diff
@@ -64,12 +64,12 @@ defmodule PlenarioWeb.Admin.MetaController do
           true)
 
         conn
-        |> put_flash(:success, "Meta #{meta.name} approved.")
+        |> put_flash(:success, "Meta #{inspect(meta.name)} approved.")
```

Comment:
Why `inspect`? If it's just to provide quotes around names (which I'm not sure is super important, but whatever) why not just `"Meta \"#{meta.name}\" approved."`?

Reply:
For this I think it's a good idea, just in case `meta.name` happens to be `nil`. Without inspect it doesn't show up and you'd see "Meta approved". At a glace there'd be nothing wrong with that message but `meta` not having a name would cause errors down the line.

Reply:
Before I was getting a nonhuman error in the flash which looks to be corrected since I can't reproduce, I can remove these